skactiveml/pool/tests/test_typi_clust.py

```diff
@@ -0,0 +1,133 @@
+import unittest
+
+import numpy as np
+from sklearn.cluster import KMeans, DBSCAN
+from skactiveml.pool._typi_clust import TypiClust
+from skactiveml.utils import MISSING_LABEL
+from skactiveml.tests.template_query_strategy import (
+    TemplateSingleAnnotatorPoolQueryStrategy,
+)
+
+
+class TestTypiClust(
+    TemplateSingleAnnotatorPoolQueryStrategy, unittest.TestCase
+):
+    def setUp(self):
+        query_default_params_clf = {
+            "X": np.linspace(0, 1, 20).reshape(10, 2),
+            "y": np.hstack([[0, 1], np.full(8, MISSING_LABEL)]),
+        }
+        super().setUp(
+            qs_class=TypiClust,
+            init_default_params={
+                "cluster_algo_dict": {"n_init": "auto", "random_state": 0}
+            },
+            query_default_params_clf=query_default_params_clf,
+        )
+
+    def test_init_param_cluster_algo(self, test_cases=None):
+        test_cases = [] if test_cases is None else test_cases
+        test_cases += [
+            (1, TypeError),
+            ("string", TypeError),
+            (None, TypeError),
+            (TypiClust, TypeError),
+            (DBSCAN, TypeError),
```

Comment:
```suggestion
            (DBSCAN, TypeError),
```
Do we now why a TypeError is raised? DBSCAN is a `ClusterMixin` so the only problem would be the `n_clusters` parameter.

Reply:
Please add a test with another working clustering algorithm, e.g., `(SpectralClustering, None)`.